test/run_test.py

```diff
@@ -884,6 +884,30 @@ def run_test_ops(test_module, test_directory, options):
 }
 
 
+PYTEST_BLOCKLIST = [
```

Comment:
Let's create a tracking issue for this, so we can plan on how to address them later.  It would be a really nice BE task to clean up things here.